src/style.css

```diff
@@ -1,3 +1,79 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  width: 1200px;
+}
+
+header {
+  background-color: #ffff;
+  width: 100%;
+  height: 60px;
+}
+
+.container {
+  display: flex;
+  background-color: #ffff;
+  justify-content: space-between;
+  margin: 0 40px;
+}
+
+nav {
+  align-items: center;
+  display: flex;
+  font-size: 12px;
+  line-height: 15px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-top: 10px;
+  margin-bottom: 10px;
```

Comment:
Be consistent with your vertical margins. Use only top or only bottom margin to avoid margin collapse issues.